### PR TITLE
[zh-cn]Sync persistent-volumes.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstream
```yaml
content/zh-cn/docs/concepts/storage/persistent-volumes.md
```

### CODE REVIEW

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -24,6 +13,17 @@ feature:
...
+title: 持久卷
+feature:
+  title: 存储编排
+  description: >
+    自动挂载所选存储系统，包括本地存储、诸如 <a href="https://aws.amazon.com/products/storage/">AWS</a>
+    或 <a href="https://cloud.google.com/storage/">GCP</a>
+    之类公有云提供商所提供的存储或者诸如 NFS、iSCSI、Ceph、Cinder 这类网络存储系统。
+content_type: concept
+weight: 20
+---
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:26]: Why this block is moved???

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -842,14 +846,14 @@ Older versions of Kubernetes also supported the following in-tree PersistentVolu
 
 旧版本的 Kubernetes 仍支持这些“树内（In-Tree）”持久卷类型：
 
-* `photonPersistentDisk` - Photon 控制器持久化盘。（v1.15 之后 **不可用**）
-* [`scaleIO`](/zh-cn/docs/concepts/storage/volumes/#scaleio) - ScaleIO 卷（v1.21 之后 **不可用**）
+* `photonPersistentDisk` - Photon 控制器持久化盘。（从 v1.15 版本开始将**不可用**）
+* [`scaleIO`](/zh-cn/docs/concepts/storage/volumes/#scaleio) - ScaleIO 卷（从 v1.21 版本开始将**不可用**）
 * [`flocker`](/zh-cn/docs/concepts/storage/volumes/#flocker) - Flocker 存储
-  （v1.25 之后 **不可用**）
+  （v1.25 版本之后将**不可用**）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:852]: Simple is beautiful. The original is better.

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -842,14 +846,14 @@ Older versions of Kubernetes also supported the following in-tree PersistentVolu
...
 * [`quobyte`](/zh-cn/docs/concepts/storage/volumes/#quobyte) - Quobyte 卷
-  （v1.25 之后 **不可用**）
+  （v1.25 版本之后将**不可用**）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:854]: Simple is beautiful.

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 4]
```diff
@@ -842,14 +846,14 @@ Older versions of Kubernetes also supported the following in-tree PersistentVolu
...
 * [`storageos`](/zh-cn/docs/concepts/storage/volumes/#storageos) - StorageOS 卷
-  （v1.25 之后 **不可用**）
+  （v1.25 之后将**不可用**）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:856]: Simple is beautiful.

**Review by windsonsea — COMMENTED** [review]
- Simple is beautiful. 
- Do not change the correct.

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/storage/persistent-volumes.md:26]: 第 16-25 行是会被后台解释为整个页面中的第一份“文档”，Hugo 将其解释为 YAML 格式，是整篇文档的元数据，不可以移动。

File: content/zh-cn/docs/concepts/storage/persistent-volumes.md [hunk 5]
```diff
@@ -468,6 +468,12 @@ cluster. However, if you want a PVC to bind to a specific PV, you need to pre-bi
 -->
 ### 预留 PersistentVolume  {#reserving-a-persistentvolume}
 
+控制平面可以在集群中[将 PersistentVolumeClaims 绑定到匹配的 PersistentVolumes](#binding)。 但是，如果你希望 PVC 绑定到特定 PV，则需要预先绑定它们。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/persistent-volumes.md:471]: PLEASE WRAP THE LONG LINES APPROPRIATELY.

**Pull request merged**